packages/oid4vci-demo-frontend/src/components/DeepLink/index.tsx

```diff
@@ -1,22 +1,43 @@
 import React from "react"
-import style from "./DeepLink.module.css"
+import styles from "./DeepLink.module.css"
 import SSIPrimaryButton from "../SSIPrimaryButton"
 import {t} from "i18next"
+import SSISecondaryButton from "../SSISecondaryButton"
 
 export interface DeepLinkProps {
   link: string
+  buttonType?: string
+  textColor?: string
   style?: React.CSSProperties
 }
 
-export default function DeepLink(props: DeepLinkProps): React.ReactElement | null {
+function generateSecondary(props: DeepLinkProps): React.ReactElement {
```

Comment:
Please include Button in the name.

Also would call it create for something like a button

Reply:
changed the name of this method to `generateSecondaryButton`